### ISSUE TITLE
run own data .  don't run

### ISSUE THREAD

**Issue by 827346462** [matterport/Mask_RCNN#2650]
2021-07-24 20:37:14.184543: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
Using TensorFlow backend.

Configurations:
BACKBONE                       resnet101
BACKBONE_STRIDES               [4, 8, 16, 32, 64]
BATCH_SIZE                     1
BBOX_STD_DEV                   [0.1 0.1 0.2 0.2]
COMPUTE_BACKBONE_SHAPE         None
DETECTION_MAX_INSTANCES        100
DETECTION_MIN_CONFIDENCE       0.7
DETECTION_NMS_THRESHOLD        0.3
FPN_CLASSIF_FC_LAYERS_SIZE     1024
GPU_COUNT                      1
GRADIENT_CLIP_NORM             5.0
IMAGES_PER_GPU                 1
IMAGE_CHANNEL_COUNT            3
IMAGE_MAX_DIM                  512
IMAGE_META_SIZE                14
IMAGE_MIN_DIM                  512
IMAGE_MIN_SCALE                0
IMAGE_RESIZE_MODE              square
IMAGE_SHAPE                    [512 512   3]
LEARNING_MOMENTUM              0.9
LEARNING_RATE                  0.001
LOSS_WEIGHTS                   {'rpn_class_loss': 1.0, 'rpn_bbox_loss': 1.0, 'mrcnn_class_loss': 1.0, 'mrcnn_bbox_loss': 1.0, 'mrcnn_mask_loss': 1.0}
MASK_POOL_SIZE                 14
MASK_SHAPE                     [28, 28]
MAX_GT_INSTANCES               100
MEAN_PIXEL                     [123.7 116.8 103.9]
MINI_MASK_SHAPE                (56, 56)
NAME                           shapes
NUM_CLASSES                    2
POOL_SIZE                      7
POST_NMS_ROIS_INFERENCE        1000
POST_NMS_ROIS_TRAINING         2000
PRE_NMS_LIMIT                  6000
ROI_POSITIVE_RATIO             0.33
RPN_ANCHOR_RATIOS              [0.5, 1, 2]
RPN_ANCHOR_SCALES              (48, 96, 192, 384, 768)
RPN_ANCHOR_STRIDE              1
RPN_BBOX_STD_DEV               [0.1 0.1 0.2 0.2]
RPN_NMS_THRESHOLD              0.7
RPN_TRAIN_ANCHORS_PER_IMAGE    256
STEPS_PER_EPOCH                100
TOP_DOWN_PYRAMID_SIZE          256
TRAIN_BN                       False
TRAIN_ROIS_PER_IMAGE           100
USE_MINI_MASK                  True
USE_RPN_ROIS                   True
VALIDATION_STEPS               50
WEIGHT_DECAY                   0.0001

0
F:\Mask_RCNN-master\Mask_RCNN-master\date\labelme_json/000001_json/img.png
1
F:\Mask_RCNN-master\Mask_RCNN-master\date\labelme_json/000002_json/img.png
2
F:\Mask_RCNN-master\Mask_RCNN-master\date\labelme_json/000003_json/img.png
0
F:\Mask_RCNN-master\Mask_RCNN-master\date\labelme_json/000001_json/img.png
1
F:\Mask_RCNN-master\Mask_RCNN-master\date\labelme_json/000002_json/img.png
2
F:\Mask_RCNN-master\Mask_RCNN-master\date\labelme_json/000003_json/img.png
WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\backend\tensorflow_backend.py:508: The name 
tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\backend\tensorflow_backend.py:68: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\backend\tensorflow_backend.py:3837: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\backend\tensorflow_backend.py:3661: The name tf.nn.max_pool is deprecated. Please use tf.nn.max_pool2d instead.

WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\backend\tensorflow_backend.py:1944: The name tf.image.resize_nearest_neighbor is deprecated. Please use tf.compat.v1.image.resize_nearest_neighbor instead.   

WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\tensorflow_core\python\ops\array_ops.py:1475: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:From f:\Mask_RCNN-master\Mask_RCNN-master\mrcnn\model.py:553: The name tf.random_shuffle is deprecated. Please use tf.random.shuffle instead.

WARNING:tensorflow:From f:\Mask_RCNN-master\Mask_RCNN-master\mrcnn\utils.py:202: The name tf.log is deprecated. Please use tf.math.log instead.

WARNING:tensorflow:From f:\Mask_RCNN-master\Mask_RCNN-master\mrcnn\model.py:600: calling crop_and_resize_v1 (from 
tensorflow.python.ops.image_ops_impl) with box_ind is deprecated and will be removed in a future version.
Instructions for updating:
box_ind is deprecated, use box_indices instead
WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\backend\tensorflow_backend.py:168: The name 
tf.get_default_session is deprecated. Please use tf.compat.v1.get_default_session instead.

WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\backend\tensorflow_backend.py:175: The name 
tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\backend\tensorflow_backend.py:180: The name 
tf.Session is deprecated. Please use tf.compat.v1.Session instead.

2021-07-24 20:37:18.571564: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
2021-07-24 20:37:18.574690: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library nvcuda.dll
2021-07-24 20:37:18.606738: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties:
name: GeForce RTX 3060 Laptop GPU major: 8 minor: 6 memoryClockRate(GHz): 1.702
pciBusID: 0000:01:00.0
2021-07-24 20:37:18.607219: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
2021-07-24 20:37:18.609993: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2021-07-24 20:37:18.612701: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_100.dll
2021-07-24 20:37:18.613942: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_100.dll
2021-07-24 20:37:18.617330: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_100.dll
2021-07-24 20:37:18.619851: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_100.dll
2021-07-24 20:37:18.627847: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2021-07-24 20:37:18.628302: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 02021-07-24 20:40:30.549373: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1159] Device interconnect StreamExecutor with strength 1 edge matrix:
2021-07-24 20:40:30.549480: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1165]      0
2021-07-24 20:40:30.549597: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1178] 0:   N
2021-07-24 20:40:30.549793: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1304] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 4753 MB memory) -> physical GPU (device: 0, name: GeForce RTX 3060 Laptop GPU, pci bus id: 0000:01:00.0, compute capability: 8.6)
WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\backend\tensorflow_backend.py:184: The name 
tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\backend\tensorflow_backend.py:193: The name 
tf.is_variable_initialized is deprecated. Please use tf.compat.v1.is_variable_initialized instead.

WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\backend\tensorflow_backend.py:200: The name 
tf.variables_initializer is deprecated. Please use tf.compat.v1.variables_initializer instead.

Starting at epoch 0. LR=0.001

Checkpoint Path: F:\Mask_RCNN-master\Mask_RCNN-master\logs\shapes20210724T2040\mask_rcnn_shapes_{epoch:04d}.h5
Selecting layers to train
fpn_c5p5               (Conv2D)
fpn_c4p4               (Conv2D)
fpn_c3p3               (Conv2D)
fpn_c2p2               (Conv2D)
fpn_p5                 (Conv2D)
fpn_p2                 (Conv2D)
fpn_p3                 (Conv2D)
fpn_p4                 (Conv2D)
In model:  rpn_model
    rpn_conv_shared        (Conv2D)
    rpn_class_raw          (Conv2D)
    rpn_bbox_pred          (Conv2D)
mrcnn_mask_conv1       (TimeDistributed)
mrcnn_mask_bn1         (TimeDistributed)
mrcnn_mask_conv2       (TimeDistributed)
mrcnn_mask_bn2         (TimeDistributed)
mrcnn_class_conv1      (TimeDistributed)
mrcnn_class_bn1        (TimeDistributed)
mrcnn_mask_conv3       (TimeDistributed)
mrcnn_mask_bn3         (TimeDistributed)
mrcnn_class_conv2      (TimeDistributed)
mrcnn_class_bn2        (TimeDistributed)
mrcnn_mask_conv4       (TimeDistributed)
mrcnn_mask_bn4         (TimeDistributed)
mrcnn_bbox_fc          (TimeDistributed)
mrcnn_mask_deconv      (TimeDistributed)
mrcnn_class_logits     (TimeDistributed)
mrcnn_mask             (TimeDistributed)
WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\optimizers.py:757: The name tf.train.Optimizer is deprecated. Please use tf.compat.v1.train.Optimizer instead.

D:\Anaconda\envs\mask\lib\site-packages\tensorflow_core\python\framework\indexed_slices.py:424: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
D:\Anaconda\envs\mask\lib\site-packages\tensorflow_core\python\framework\indexed_slices.py:424: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
D:\Anaconda\envs\mask\lib\site-packages\tensorflow_core\python\framework\indexed_slices.py:424: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\backend\tensorflow_backend.py:977: The name 
tf.assign_add is deprecated. Please use tf.compat.v1.assign_add instead.

WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\backend\tensorflow_backend.py:964: The name 
tf.assign is deprecated. Please use tf.compat.v1.assign instead.

WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\callbacks.py:783: The name tf.summary.merge_all is deprecated. Please use tf.compat.v1.summary.merge_all instead.

WARNING:tensorflow:From D:\Anaconda\envs\mask\lib\site-packages\keras\callbacks.py:786: The name tf.summary.FileWriter is deprecated. Please use tf.compat.v1.summary.FileWriter instead.

Epoch 1/50
image_id 1
f:/Mask_RCNN-master/Mask_RCNN-master/train.py:88: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  temp = yaml.load(f.read())
[1]
D:\Anaconda\envs\mask\lib\site-packages\skimage\transform\_warps.py:830: FutureWarning: Input image dtype is bool. Interpolation is not defined with bool data type. Please set order to 0 or explicitely cast input image to another data type. Starting from version 0.19 a ValueError will be raised instead of this warning.
  order = _validate_interpolation_order(image.dtype, order)
2021-07-24 20:40:39.773231: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2021-07-24 20:41:38.927370: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll

**Comment by 827346462** [comment]
always show this.

**Comment by 827346462** [comment]
2021-07-24 20:53:09.808358: W tensorflow/stream_executor/cuda/redzone_allocator.cc:312] Internal: Invoking ptxas not supported on Windows
Relying on driver to perform ptx compilation. This message will be only logged once.
2021-07-24 20:53:12.435610: E tensorflow/stream_executor/cuda/cuda_blas.cc:428] failed to run cuBLAS routine: CUBLAS_STATUS_EXECUTION_FAILED
2021-07-24 20:53:12.439062: I tensorflow/stream_executor/stream.cc:4925] [stream=000001DE705644F0,impl=000001DE64E64E30] did not memcpy device-to-host; source: 000000083802EC00
2021-07-24 20:53:12.439235: W tensorflow/core/framework/op_kernel.cc:1651] OP_REQUIRES failed at conv_grad_input_ops.cc:1161 : Not found: No algorithm worked!
2021-07-24 20:53:12.439371: I tensorflow/stream_executor/stream.cc:4963] [stream=000001DE705644F0,impl=000001DE64E64E30] did not memzero GPU location; source: 000000DB897DDE78
2021-07-24 20:53:12.439439: I tensorflow/stream_executor/stream.cc:5418] [stream=000001DE705644F0,impl=000001DE64E64E30] Internal: stream did not block host until done; was already in an error state
2021-07-24 20:53:12.439746: I tensorflow/stream_executor/stream.cc:4976] [stream=000001DE705644F0,impl=000001DE64E64E30] did not memset GPU location; source: 000000DB8901C0C8; size: 8388608; pattern: ffffffff
2021-07-24 20:53:12.440064: I tensorflow/stream_executor/stream.cc:316] did not allocate timer: 000000DB897DDE20  
2021-07-24 20:53:12.440244: W tensorflow/core/kernels/gpu_utils.cc:65] Failed to check cudnn convolutions for out-of-bounds reads and writes with an error message: 'stream did not block host until done; was already in an error state'; skipping this check. This only means that we won't check cudnn for out-of-bounds reads and writes. This message will only be printed once.
2021-07-24 20:53:12.440321: I tensorflow/stream_executor/stream.cc:4976] [stream=000001DE705644F0,impl=000001DE64E64E30] did not memset GPU location; source: 000000DB8901C0E8; size: 8388608; pattern: ffffffff
2021-07-24 20:53:12.440414: I tensorflow/stream_executor/stream.cc:1964] [stream=000001DE705644F0,impl=000001DE64E64E30] did not enqueue 'start timer': 000000DB897DDE20
2021-07-24 20:53:12.440692: I tensorflow/stream_executor/stream.cc:4976] [stream=000001DE705644F0,impl=000001DE64E64E30] did not memset GPU location; source: 000000DB89F9C668; size: 8388608; pattern: ffffffff
2021-07-24 20:53:12.440800: W tensorflow/core/framework/op_kernel.cc:1651] OP_REQUIRES failed at conv_grad_input_ops.cc:1161 : Not found: No algorithm worked!
2021-07-24 20:53:12.440895: I tensorflow/stream_executor/stream.cc:1976] [stream=000001DE705644F0,impl=000001DE64E64E30] did not enqueue 'stop timer': 000000DB897DDE20
2021-07-24 20:53:12.441016: I tensorflow/stream_executor/stream.cc:4976] [stream=000001DE705644F0,impl=000001DE64E64E30] did not memset GPU location; source: 000000DB89F9C688; size: 8388608; pattern: ffffffff
2021-07-24 20:53:12.441143: I tensorflow/stream_executor/stream.cc:4963] [stream=000001DE705644F0,impl=000001DE64E64E30] did not memzero GPU location; source: 000000DB8901DE98
2021-07-24 20:53:12.441244: F tensorflow/stream_executor/gpu/gpu_timer.cc:65] Check failed: start_event_ != nullptr && stop_event_ != nullptr
2021-07-24 20:53:12.441411: W tensorflow/core/framework/op_kernel.cc:1651] OP_REQUIRES failed at conv_grad_input_ops.cc:1161 : Not found: No algorithm worked!

**Comment by 827346462** [comment]
![image](https://user-images.githubusercontent.com/50097546/126869082-4b07d50b-9205-4af6-9a2f-6091187358c3.png)
logs show.

**Comment by caofei996** [comment]
see it  https://stackoverflow.com/questions/62330374/input-image-dtype-is-bool-interpolation-is-not-defined-with-bool-data-type
